hermes/queues/send-new-moderator-email.js

```diff
@@ -0,0 +1,38 @@
+// @flow
+const debug = require('debug')('hermes:queue:send-new-moderator-welcome-email');
+import Raven from 'shared/raven';
+import sendEmail from '../send-email';
+import {
+  SEND_NEW_MODERATOR_WELCOME_EMAIL,
+  NEW_MODERATOR_WELCOME_TEMPLATE,
+} from './constants';
+import type { Job, SendAddedModeratorEmailJobData } from 'shared/bull/types';
+
+export default (job: Job<SendAddedModeratorEmailJobData>) => {
+  debug(`\nnew job: ${job.id}`);
+  const { recipient, community } = job.data;
+  debug(`\nsending notification to user: ${recipient.email}`);
+
+  const subject = `You are now a moderator of the ${community.name} community!`;
+  const preheader = ``; //TODO(@littlestudent)
```

Comment:
not sure what to use as the preheader.
suggestions?

Reply:
How about
Subject: "You have been added as a moderator in the ${community.name} community on Spectrum"
Preheader: "Go to the ${community.name} community to get started with moderation"